tests/langchain/test_langchain_autolog.py

```diff
@@ -0,0 +1,224 @@
+import os
+import re
+from unittest import mock
+
+from langchain.chains import LLMChain
+from langchain.llms import OpenAI
+from langchain.prompts import PromptTemplate
+
+import mlflow
+from mlflow import MlflowClient
+from mlflow.models import Model
+from mlflow.models.signature import infer_signature
+from mlflow.models.utils import _read_example
+from mlflow.utils.openai_utils import (
+    TEST_CONTENT,
+    _mock_chat_completion_response,
+    _mock_request,
+)
+
+MODEL_DIR = "model"
+TEST_CONTENT = "test"
+MLFLOW_CALLBACK_METRICS = [
```

Comment:
Are these defined anywhere in langchain so that cross version tests can just read from the list of provided constants from within their library (to simplify our tests if they end up changing these and to make sure that we don't miss new ones in the future)

Reply:
They're not defined inside langchain but we can add it into the file of MLflowCallbackHandler :D